### DISCUSSION TITLE
how to build opentelemetry-cpp-contrib just like the release?

### DISCUSSION THREAD

**Discussion by zhuizhuhaomeng** [open-telemetry/community#1323] [Community Discussions]
I would like to build the  opentelemetry-cpp-contrib just like the release? 
Is there any document about this ?
https://github.com/open-telemetry/opentelemetry-cpp-contrib/releases/tag/webserver%2Fv1.0.3

**Comment by zhuizhuhaomeng** [comment]
@kpratyus would you please share with us how you build the package?

**Reply by DebajitDas** [reply]
@zhuizhuhaomeng 
If you want build ```otel-webserver-module```, you need to navigate to ```instrumentation/otel-webserver-module``` and trigger a build as mentioned [here](https://github.com/open-telemetry/opentelemetry-cpp-contrib/tree/main/instrumentation/otel-webserver-module#generate-the-artifact-manually-1)

There are other components inside ```exporter``` or ```instrumentation``` directory, which needs to be built separately, if you need them. The build steps are mentioned in the respective directory.

**Reply by zhuizhuhaomeng** [reply]
thank you @DebajitDas
